demo/src/pages/index.js

```diff
@@ -215,14 +215,17 @@ const IndexPage = ({ data }) => {
 
         <section>
           <h2>Feedback</h2>
-          <Feedback
-            onPositiveClick={() => {
-              alert('nice');
-            }}
-            onNegativeClick={() => {
-              alert('meh');
-            }}
-          />
+          <div
+            css={css`
+              width: 350px;
+            `}
+          >
+            <Feedback
+              onSubmit={({ sentiment, comment }) => {
```

Comment:
💯 I _really_ like this API and how you've renamed the thumbs up/down as `sentiment` here. Nice work!!